modules/jaggery-apps/portal/gadgets/user_auth_apps/js/gadget.js

```diff
@@ -15,10 +15,18 @@ function drawPage() {
     if (json != null) {
 
         if (isArray(json.return)) {
+
             for (var i in json.return) {
+
+                var username = json.return[i].username;
+
+                if (username.indexOf("carbon.super") > -1) {
```

Comment:
Since this is a Javascript, constants are not available. We only need to remove the tenant domain name if it is only the super tenant, hence checking for the "carbon.super".